content/docs/concepts/security/index.md

```diff
@@ -239,6 +271,41 @@ Istio provides two types of authentication:
   original client making the request as an end-user or device. Istio
   supports authentication with JSON Web Token (JWT) validation.
 
+### Mutual TLS authentication
+
+Service-to-service communication is tunneled through the client side [Envoy](https://envoyproxy.github.io/envoy/) and the server side Envoy.
+For a client to call a server, the process can be described as the following:
+
+1. The outbound traffic from a client is rerouted to its local sidecar Envoy.
```

Comment:
Please change the passive voice to active voice. Passive voice is a lot harder to understand for non-native speakers, it is harder to translate, and adds difficulty and complexity to the content. I suggest:
1.  Istio re-routes the outbound traffic from a client to the client's local sidecar Envoy.